README.md

```diff
@@ -188,11 +188,11 @@ The first step is to [register a new Github App] on the GitHub web site.
 
 The *Github App name*, *description*, and *homepage URL* are irrelevant, though I suggest pointing the homepage at the dashboard page.
 
-The *user authorization callback URL* should be at `<dashboard url>/auth/github/callback`.
+The *user authorization callback URL* should be at `<<DASHBOARD URL>>/auth/github/callback`.
```

Comment:
I don't think it needs to be double quote and shouting.

Reply:
I changed this after realizing that I was using a different interpolation syntax in different parts of the README (notice the spot where `https://github.com/apps/your-app-name` is changed to `https://github.com/apps/<<APP NAME>>`).

The reason for going with shouting and double-quote, of course, is so that it doesn't look like a part of the URL. I figure there's no real harm in being as unsubtle as you can short of doing real harm (two quotes are fine, two hundred would break the text layout).